man/docker-daemon.8.md

```diff
@@ -0,0 +1,402 @@
+% DOCKER(1) Docker User Manuals
+
+% William Henry 
```

Comment:
If we're talking about the changes; not sure what name should be here (perhaps yours, given that it's a new file), idk :smile: